{"uuid": "55e915c2-e63c-43d4-b170-38407eb21fc8", "children": ["424ca849-c8dc-4ed6-86dd-8d06d2c08d82"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 128\nCurrent browser version is 130.0.6723.70 with binary path C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\n\tGetHandleVerifier [0x00007FF6C270B632+29090]\n\t(No symbol) [0x00007FF6C267E6E9]\n\t(No symbol) [0x00007FF6C253B1CA]\n\t(No symbol) [0x00007FF6C257D6B2]\n\t(No symbol) [0x00007FF6C257C751]\n\t(No symbol) [0x00007FF6C257729C]\n\t(No symbol) [0x00007FF6C2571E0A]\n\t(No symbol) [0x00007FF6C25C236D]\n\t(No symbol) [0x00007FF6C25C1A20]\n\t(No symbol) [0x00007FF6C25B6493]\n\t(No symbol) [0x00007FF6C25809B1]\n\t(No symbol) [0x00007FF6C2581B11]\n\tGetHandleVerifier [0x00007FF6C2A2883D+3294125]\n\tGetHandleVerifier [0x00007FF6C2A74423+3604371]\n\tGetHandleVerifier [0x00007FF6C2A6A2E7+3563095]\n\tGetHandleVerifier [0x00007FF6C27C6F16+797318]\n\t(No symbol) [0x00007FF6C268986F]\n\t(No symbol) [0x00007FF6C2685454]\n\t(No symbol) [0x00007FF6C26855E0]\n\t(No symbol) [0x00007FF6C2674A7F]\n\tBaseThreadInitThunk [0x00007FFA7EE87374+20]\n\tRtlUserThreadStart [0x00007FFA7F21CC91+33]\n\n", "trace": "  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\Desktop\\Diplom\\Diplom_3\\conftest.py\", line 24, in driver\n    browser = webdriver.Chrome()\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Влада\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1730212242428, "stop": 1730212247704}], "start": 1730212242428, "stop": 1730212247830}